test/test-pages/yahoo-4/expected-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "トレンドマイクロ、公衆無線LANを安全に使うためのアプリ「フリーWi-Fiプロテクション」 （CNET Japan） - Yahoo!ニュース",
+  "byline": null,
```

Comment:
Can you file issues about the missing byline and about us not stripping the "Yahoo!" bit at the end from the title?

Other than that, this looks OK to me. I'm assuming you checked there aren't major issues with the content we read otherwise.